waku/v2/node/wakunode2.nim

```diff
@@ -573,11 +584,40 @@ proc mountRelay*(node: WakuNode,
+      var groupSize = int(groupKeys.len/2)
+
+      debug "rln-relay membership index", rlnRelayMemIndex
+
+      # validate the user-supplied membership index
+      if rlnRelayMemIndex < uint(0) or rlnRelayMemIndex >= uint(groupSize):
+        error "wrong membership index, failed to mount WakuRLNRelay"
+      else: 
+        # prepare group related inputs from the hardcoded keys
+        var groupKeyPairs = groupKeys.toMembershipKeyPairs()
+        var groupIDCommitments = groupKeyPairs.mapIt(it.idCommitment)
+
+        # mount rlnrelay in offline mode
+        waitFor node.mountRlnRelay(groupOpt= some(groupIDCommitments), memKeyPairOpt = some(groupKeyPairs[rlnRelayMemIndex]), memIndexOpt= some(rlnRelayMemIndex), onchainMode = false)
```

Comment:
Could these arguments be constructed in a util function, perhaps outside the `wakunode2` module (e.g. in `waku_setup`, or similar)? I'm worried that the `wakunode2` module may become unwieldy, especially since this is the entry point when debugging issues.

Reply:
Can you please elaborate on the use of a util func to construct arguments?

Reply:
I meant something like the [`setupNat` function](https://github.com/status-im/nim-waku/blob/dcba72677e472e93b03c17d22e1ca7d427d953c7/waku/v2/node/wakunode2.nim#L856) where the arguments (e.g. groupOpt, memKeyPairOpt) are constructed in a util function in a separate module to keep `wakunode2` relatively uncluttered. However, now that `StaticGroupKeys` live elsewhere it's already much clearer and probably not necessary. Thanks! :)

Reply:
Thanks for the explanation! 
I'll certainly use a util function in case that the input preparation gets more complicated (in the future versions), thanks!